.github/workflows/npm-release.yml

```diff
@@ -40,8 +45,18 @@ jobs:
 
       - run: yarn install --immutable
 
-      - run: yarn lerna version ${{ inputs.version }} --no-private --conventional-commits --create-release github --generate-release-notes --y
-
-      - run: |
-          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
-          yarn run publish --y
+      - name: Version and publish
+        run: |
+          if [ "${{ inputs.version }}" == "custom" ]; then
+            if [ -z "${{ inputs.custom_version }}" ]; then
+              echo "Error: Custom version is required when 'custom' is selected"
+              exit 1
+            fi
+            yarn lerna version ${{ inputs.custom_version }} --no-git-tag-version --yes --force-publish
```

Comment:
It's because lerna does not do anything if there is no commit difference: https://lerna.js.org/docs/features/version-and-publish#synchronized-versions

Reply:
Right thanks